org.eclipse.scout.rt.server.commons/src/main/java/org/eclipse/scout/rt/server/commons/authentication/TrivialAccessController.java

```diff
@@ -65,6 +68,14 @@ public boolean handle(final HttpServletRequest request, final HttpServletRespons
       return false;
     }
 
+    if (isUnloadRequest(request) && !isAuthenticatedRequest(request)) {
+      // Unload requests are handled by the {@code UnloadRequestHandler}.
+      // The unload request is supposed to close a UI session by the browser. If the current session is not authenticated then there is no session to unload.
+      // Answering the unauthenticated unload request here prevents saml or oidc filters to produce unwanted redirect urls after authentication (we want to avoid the /unload/ url as redirect url).
```

Comment:
```suggestion
      // Answering the unauthenticated unload request here prevents other controllers (e.g. saml or oidc) to produce unwanted redirect urls after authentication (we want to avoid the /unload/ url as redirect url).
```

Use a more general wording here. SAML or OIDC are entirely implementation-dependent and not part of the Scout platform.

Reply:
Log message changed as recommended.